src/mp_ui_alerts.hpp

```diff
@@ -42,7 +42,4 @@ namespace mp::ui_alerts {
 	bool get_def_pref_sound(const std::string &);
 	bool get_def_pref_notif(const std::string &);
 	bool get_def_pref_lobby(const std::string &);
-
-	// Note, this list of items must match those ids defined in data/gui/dialogs/mp_alerts_options.cfg
```

Comment:
This seems like a useful comment that should rather be updated and not removed.

Reply:
I'll re-add it.